libraries/AP_NavEKF3/AP_NavEKF3_PosVelFusion.cpp

```diff
@@ -429,6 +429,29 @@ void NavEKF3_core::CorrectGPSForAntennaOffset(gps_elements &gps_data) const
     gps_data.hgt += posOffsetEarth.z;
 }
 
+/*
+  correct Baro data for position offset relative to the IMU
+ */
+void NavEKF3_core::CorrectBaroOffset(baro_elements &baro_data) const
```

Comment:
```suggestion
void NavEKF3_core::CorrectBaroPositionOffset(baro_elements &baro_data) const
```

"offset" can mean many things